docs/client/README.md

```diff
@@ -0,0 +1,10 @@
+# Semantic Conventions For Client SDKs
+
+**Status**: [Experimental][DocumentStatus]
+
+This document defines general semantic conventions for client-side applications. Client applications are differentiated 
+from server applications in that they service a user rather than requests. This adds unique requirements and needs to 
+client-side semantic conventions. 
```

Comment:
Suggest adding a sentence to anchor the abstract concept of "client" into the common practical types.

```suggestion
This document defines general semantic conventions for client-side applications. 
The most common types of client applications are web applications (that run in a web browser)
and mobile apps (that run on a phone or tablet).
Client applications are differentiated from server applications in that they service a user rather than requests. This adds unique requirements and needs to client-side semantic conventions. 
```